configs/16.0/release_notes/release_notes-inst.html

```diff
@@ -320,8 +349,41 @@ <h3>Using aptitude or apt</h3>
                                 <p>Aptitude and apt support package upgrades for new revision releases (i.e. 10.0-0 to 10.0-1).  For new major releases, install as if a new installation.</p>
 endif_ubuntu-20 -->
 
+<!-- if_ubuntu-22
+                        <p>The gpg public key <span class="code">615d762f.gpg.key</span> will be provided in the repository where these release notes were found.  The public key can be used to verify the authenticity of both the Advance Toolchain install packages and the repository contents. It can be imported as follows:</p>
+                        <p class="code">wget -qO- https://public.dhe.ibm.com/software/server/POWER/Linux/toolchain/at/ubuntu/dists/jammy/615d762f.gpg.key | sudo tee -a /etc/apt/trusted.gpg.d/615d762f.asc</p>
```

Comment:
The guidelines for Ubuntu 20 use `apt-key add`. Any reason to not continue using it here?

Reply:
This says to to _not_ just add the key to `/etc/apt/trusted.gpg.d`, but instead to modify the repository definition in `sources.list` (or equivalent): https://askubuntu.com/questions/1286545/what-commands-exactly-should-replace-the-deprecated-apt-key

Reply:
I added this in the patch.